ddtrace/tracer/textmap.go

```diff
@@ -322,6 +359,24 @@ func (p *propagator) extractTextMap(reader TextMapReader) (ddtrace.SpanContext,
 	return &ctx, nil
 }
 
+func extractPropagatingTags(ctx *spanContext, v string, err error) {
+	if ctx.trace == nil {
+		ctx.trace = newTrace()
+	}
+	ctx.trace.mu.Lock()
+	defer ctx.trace.mu.Unlock()
+	if len(v) > propagationExtractMaxSize {
+		log.Warn("did not extract trace tags, longer than max size limit: %d", propagationExtractMaxSize)
```

Comment:
```suggestion
		log.Warn("Did not extract trace tags, size limit exceeded: %d", propagationExtractMaxSize)
```
What is the actual impact on the user here? Should "trace tags" mean anything to them? This seems more like an internal term. If there is an actual impact, let's refer specifically to that. If there is no impact on the user, then let's not warn anything.